fpsim/base.py

```diff
@@ -405,6 +412,14 @@ def npts(self):
         except:
             return 0
 
+    @property
+    def tperyear(self):
```

Comment:
Oh I see, i completely missed this the first time around. Can I be annoying and suggest a different name?

- `tiperyear` -- we've been using `ti` as a reference to timesteps, as opposed to  just `t` that usually refers to a float representation of time (in years, months, etc) 
- `n_tpy` or `ntpy` (stands for number of timepoints per year) if we go for a very short variable name like `npts`
- `timepoints_per_year` maybe too on the nose :grin: